BookReader/BookReader.js

```diff
@@ -4138,7 +3429,7 @@ BookReader.prototype.initToolbar = function(mode, ui) {
           offCanvas: {
             wrapPageIfNeeded: false,
             zposition: 'next',
-            pageSelector: '#BookReader',
+            pageSelector: this.el,
```

Comment:
Suggestion: Call this a selector since it’s a string (or actually store a DOM/jQuery element in it).

Reply:
I tried changing the setup function to use `$(this.el)`, to have jQuery turn it into an element, but it caused some issues. So I think this should be made into it's own ticket and fixed later.